src/components/Tweet.js

```diff
@@ -1,37 +1,77 @@
-function Tweet() {
+import ProfileImage from "./ProfileImage";
+import User from "./User";
+import Timestamp from "./Timestamp";
+import Message from "./Message";
+import Actions from "./Actions";
+
+function Tweet(props) {
```

Comment:
Instead of repeating the code for the number of tweets you have, you can just pass the generic function using `props`, and then, on the `App.js`, you could just repeat the number of tweets you want to display.

```
function Tweet(props) {
  return (
    <div className="tweet">
      <ProfileImage image={props.tweet.user.image} />

      <div className="body">
        <div className="top">
          <User userData={props.tweet.user} />

          <Timestamp time={props.tweet.timestamp} />
        </div>

        <Message message={props.tweet.message} />

        <Actions />
      </div>

      <i className="fas fa-ellipsis-h"></i>
    </div>
  );
}
```